content/backmatter.tex

```diff
@@ -334,6 +334,7 @@ \section{Overview}\label{subsec:dep_overview}
     & \shortstack{\textbf{Last Version Supported}}
     & \textbf{Replaced By} \\
     \hline
+    Header Directory: \hyperref[subsec:dep_rationale:mpp]{\HEADER{mpp}} & 1.1 & Current & (none) \\ \hline
```

Comment:
To be consistent with the other deprecations, should we list this as ```\CorCpp: \HEADER{mpp/shmem.h}``` and ```\Fortran: \HEADER{mpp/shmem.fh}```?

Also, what about the extensions headers?  Should we also list those here for completeness?